qvisor/src/vmspace/nvidia.rs

```diff
@@ -135,6 +136,30 @@ pub fn NvidiaProxy(cmd: ProxyCommand, parameters: &ProxyParameters) -> Result<i6
             // );
             return Ok(ret as i64);
         }
+        ProxyCommand::CudaUnRegisterFatBinary => {
+            let moduleKey = parameters.para1;
+            //error!("unregister module key:{:x}", moduleKey);
+
+            let mut module = match MODULES.lock().get(&moduleKey){
+                Some(module) => {
+                    //error!("module: {:x} for this module key: {:x} has been found",module, moduleKey);
+                    *module 
+                }
+                None => {
+                    //error!("no module be found with this fatcubinHandle:{:x}",moduleKey);
+                    0
+                }
+            };
+            let ret = unsafe{
+                cuda_driver_sys::cuModuleUnload((module as *const u64) as  CUmodule) 
+            };
+            //error!("ret: {:?}", ret);
+            // delete the module 
+            MODULES.lock().remove(&moduleKey);
```

Comment:
We need to figure out a way to remove items from other two treemaps as well. If cannot do it now, add a todo comment here to work on it later.

Reply:
have add a comment